products/api-platform.md

```diff
@@ -78,7 +79,7 @@ releases:
 
 > [API Platform](https://api-platform.com/) is a full stack framework dedicated to API-driven projects and implementing the Jamstack architecture.
 
-New versions are released "when they are ready". Releases follow [Semantic Versioning](https://semver.org/). Besides the development branch, two versions are maintained:
+Releases follow [Semantic Versioning](https://semver.org/). A new minor version is released every six months, and a new major version very two years along with a minor version that brings an upgrade path. Besides the development branch, two versions are maintained:
```

Comment:
i thing there is a minor typo here
`version very two years`
very -> every

Reply:
This is very true. I'm fixing this tomorrow and put the PR RFR :)